homework/content.js

```diff
@@ -1,37 +1,42 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+function Content() {
+	this._wrapper = document.querySelector('#note-content-wrapper');
+}
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+Content.prototype = {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+	start : function () {
+		window.addEventListener('note-open', (function(event) {
+		var note = event.detail;
+		this.resetWrapper();
+		this.drawNote(note);
+		}).bind(this));
+	},
+	resetWrapper : function () {
+		this._wrapper.innerHTML = '';
+	},
 
-  function drawNote(note) {
-    var title = note.title;
-    var h = document.createElement('h2');
-    h.textContent = title;
-    var passages = note.passages;
-    var buff = document.createDocumentFragment();
-    passages.forEach(function(passage) {
-      var p = document.createElement('p');
-      p.classList.add('note-passage');
-      p.textContent = passage;
-      buff.appendChild(p);
-    });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
-  }
-
-  document.addEventListener('DOMContentLoaded', function(event) {
-    start();
-  });
-})();
+	drawNote : function (note) {
+		var title = note.title;
+		var h = document.createElement('h2');
+		h.textContent = title;
+		var passages = note.passages;
+		var buff = document.createDocumentFragment();
+		passages.forEach(function(passage) {
+			  var p = document.createElement('p');
+			  p.classList.add('note-passage');
+			  p.textContent = passage;
+			  buff.appendChild(p);
+		});
+		this._wrapper.appendChild(h);
+		this._wrapper.appendChild(buff);
+	},
+	addition: function (a,b) {
+	    return a + b;
+	}
+};
+var content = new Content();
+document.addEventListener('DOMContentLoaded', function(event) {
```

Comment:
It's better to put all startup code like this in one method, rather than two native handlers like this.